src/app/containers/Include/amp/VjAmp.jsx

```diff
@@ -7,6 +7,39 @@ const IncludeGrid = styled(GridItemConstrainedMedium)`
   display: grid;
 `;
 
+const PocketTapArea = styled.div`
+  &&& {
+    display: block;
+    height: 70px;
+    background: linear-gradient(
+      0deg,
+      #fff 0%,
+      rgba(255, 255, 255, 0.816947) 66%,
+      rgba(255, 255, 255, 0.0718488) 100%
+    );
+    &:after {
+      display: inline-block;
+      content: '';
+      width: 100vw;
+      border-bottom: 2px solid #1380a1;
+      transform: translateY(2.4rem);
+    }
```

Comment:
Where did these styles come from are they ported from legacy amp implementation? Have we confirmed with UX that this styling is what we want?

Reply:
UX do not have designs on the show more button, but yes am in conversation with them to get the stylings they want.